packages/solid/components/Column/Grid.types.ts

```diff
@@ -31,15 +32,29 @@ export interface ColumnProps extends UIComponentProps, ColumnStyleProperties {
   /** function to be called on up click */
   onUp?: KeyHandler;
 
+  /** function to be called on down click */
+  onLeft?: KeyHandler;
+
+  /** function to be called on up click */
+  onRight?: KeyHandler;
+
```

Comment:
extremely minor update

```suggestion
  /** function to be called on left click */
  onLeft?: KeyHandler;

  /** function to be called on right click */
  onRight?: KeyHandler;

```